features/support/env.js

```diff
@@ -42,44 +42,33 @@ module.exports = function () {
         this.OSRM_PORT = process.env.OSRM_PORT && parseInt(process.env.OSRM_PORT) || 5000;
         this.HOST = 'http://127.0.0.1:' + this.OSRM_PORT;
 
-        // TODO make sure this works on win
-        if (process.platform.match(/indows.*/)) {
+        if (process.platform.match(/^win.*/)) {
             this.TERMSIGNAL = 9;
             this.EXE = '.exe';
-            this.LIB = '.dll';
-            this.QQ = '"';
         } else {
             this.TERMSIGNAL = 'SIGTERM';
             this.EXE = '';
+        }
 
-            // heuristically detect .so/.a suffix
-            this.LIB = null;
-
-            try {
-                const dot_a = util.format('%s/libosrm%s', this.BIN_PATH, '.a');
-                fs.accessSync(dot_a, fs.F_OK);
-                this.LIB = '.a';
-            } catch(e) { /*nop*/ }
-
+        // heuristically detect .so/.a/.dll/.lib suffix
+        this.LIB = ['lib%s.a', 'lib%s.so', '%s.dll', '%s.lib'].find((format) => {
```

Comment:
here instead of library extensions a library platform-dependent format string is used  -> windows libraries have no `lib` prefix